developer-docs-site/docs/guides/move-guides/guide-move-transactional-testing.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Guide for Move Transactional Testing"
+slug: "guide-move-transactional-testing"
+---
+
```

Comment:
@jjleng given that this is a new feature which we're not confident about its long term viability, we should probably preface this with a note that it is exploratory and our expansion and support for it is dependent on community usage.

we should also look at the telemetry for the cli and make sure we gather signal

Reply:
Sure, will add a note.